omegaconf/_utils.py

```diff
@@ -191,15 +191,20 @@ def _resolve_optional(type_: Any) -> Tuple[bool, Any]:
     return False, type_
 
 
-def _is_optional(obj: Any, key: Optional[Union[int, str]] = None) -> bool:
+def _is_optional(obj: Any, key: Optional[Union[int, str]] = _DEFAULT_MARKER_) -> bool:
```

Comment:
Why use _DEFAULT_MARKER_ here? I don't think we support None keys.

Reply:
I was thinking that we might support them some time in the future...

Reply:
Changed it back to `None` in https://github.com/omry/omegaconf/pull/705/commits/c20409e455a556c69a94ed8600d691f51fce8c94.